### ISSUE TITLE
[3.2] A minor mathematical mistake in 3.2 Deutsch-Jozsa Algorithm

### ISSUE THREAD

**Issue by Chen-Jialin** [qiskit-community/qiskit-textbook#1019]
In 3.2 Deutsch-Jozsa Algorithm > 1.3 Quantum Solution, Step 5 of the algorithm, it write:
```
the probability of measuring \lvert0\rangle^{\otimes n} = |\frac{1}{\sqrt{2^n}}\sum_{x=0}^{2^n-1}(-1)^{f(x)}|^2
```
However, the correct probability should be
```
|\frac{1}{2^n}\sum_{x=0}^{2^n-1}(-1)^{f(x)}|^2
```

**Comment by divshacker** [comment]
May I work on this @Chen-Jialin ?

**Comment by frankharkins** [comment]
Closed via #1019
